src/libcollections/str.rs

```diff
@@ -161,6 +161,9 @@ enum DecompositionType {
 /// External iterator for a string decomposition's characters.
 ///
 /// For use with the `std::iter` module.
+#[allow(deprecated)]
+#[deprecated(reason = "use the crates.io `unicode-decomp` library instead",
+             since = "1.0.0-nightly-20150415")]
```

Comment:
These `since` tags should all be "1.0.0" for now (e.g. they're applicable for the 1.0.0 release)